flaps/actions.go

```diff
@@ -0,0 +1,91 @@
+package flaps
+
+type flapsAction int
+
+const (
+	appCreate flapsAction = iota
```

Comment:
These seem like they'll be hard to maintain. Would something like the stringer generator work so code gen handles the grunt work? https://pkg.go.dev/golang.org/x/tools/cmd/stringer